packages/docusaurus-theme-translations/locales/hu/theme-common.json

```diff
@@ -0,0 +1,70 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "Bezárás",
+  "theme.BackToTopButton.buttonAriaLabel": "Vissza a tetejére",
+  "theme.CodeBlock.copied": "Másolva",
+  "theme.CodeBlock.copy": "Másolás",
+  "theme.CodeBlock.copyButtonAriaLabel": "Másolás a vágólapra",
+  "theme.CodeBlock.wordWrapToggle": "Szótörés váltása",
+  "theme.DocSidebarItem.toggleCollapsedCategoryAriaLabel": "Összecsukás/kibontás kategória '{label}'",
+  "theme.ErrorPageContent.title": "Hiba történt a oldalon.",
+  "theme.ErrorPageContent.tryAgain": "Próbáld újra",
+  "theme.NavBar.navAriaLabel": "Fő",
+  "theme.NotFound.p1": "Sajnos nem találtuk azt az oldalt, amit kerestél.",
+  "theme.NotFound.p2": "Kérjük, lépj kapcsolatba az oldal tulajdonosával, ahonnan erre a hivatkozásra léptél, hogy jelezd, hogy a hivatkozás nem működik.",
+  "theme.NotFound.title": "Az oldal nem található",
+  "theme.TOCCollapsible.toggleButtonLabel": "Ezen az oldalon tartalomjegyzék",
+  "theme.admonition.caution": "vigyázat",
+  "theme.admonition.danger": "veszély",
+  "theme.admonition.info": "információ",
+  "theme.admonition.note": "megjegyzés",
+  "theme.admonition.tip": "tanács",
+  "theme.blog.archive.description": "Archívum",
+  "theme.blog.archive.title": "Archívum",
+  "theme.blog.paginator.navAriaLabel": "Bloglista oldalának navigációja",
+  "theme.blog.paginator.newerEntries": "Újabb bejegyzések",
+  "theme.blog.paginator.olderEntries": "Régebbi bejegyzések",
+  "theme.blog.post.paginator.navAriaLabel": "Blogbejegyzés oldalának navigációja",
+  "theme.blog.post.paginator.newerPost": "Újabb bejegyzés",
+  "theme.blog.post.paginator.olderPost": "Régebbi bejegyzés",
+  "theme.blog.post.plurals": "{count} bejegyzés|{count} bejegyzés|{count} bejegyzés",
```

Comment:
Hungarian has only 2 cardinal plural forms:
https://www.unicode.org/cldr/cldr-aux/charts/34/supplemental/language_plural_rules.html

<img width="1844" alt="CleanShot 2023-02-15 at 20 04 17@2x" src="https://user-images.githubusercontent.com/749374/219127622-08282d0f-dea4-4892-9c5e-9423839b01b5.png">

Even if you did use 3 forms in this string, only the first 2 will be used in practice, in the following order: "one|other" 

I don't know why you used 3 forms exactly, but this is likely a mistake. I don't speak Hungarian and can't fix that mistake unfortunately 😅